src/components/FilterBar/FilterDisplay/Inputs/RelativeDateRangeInput.react.js

```diff
@@ -0,0 +1,32 @@
+import { RangeInput } from "./RangeInput.react";
+
+export class RelativeDateRangeInput extends RangeInput {
+  render() {
+    return (
+      <li>
+        <div className="row">
+          <div className="col-xs-6">
+            From (Today):
+            <input
+              className="form-control"
+              onBlur={this.onBlur.bind(this)}
+              onChange={this.onChange.bind(this)}
+              placeholder="from"
+              value={this.state.value.from}
+            />
```

Comment:
Wondering if exposing these inputs as free text may be confusing for users? Would a `<select>` with options for each offset be more explanatory? e.g.
- ...
- `3 days ago`
- `2 days ago`
- `Yesterday`
- `Today`
- `Tomorrow`
- `2 days from now`
- `3 days from now`
- ...

Reply:
@dn-readytech I'm planning on adding the options, but users still need the ability to enter a "custom period", don't they?